Library/Formula/rust.rb

```diff
@@ -44,7 +44,6 @@ def install
     args = ["--prefix=#{prefix}"]
     args << "--disable-rpath" if build.head?
     args << "--enable-clang" if ENV.compiler == :clang
-    args << "--release-channel=beta" unless build.head?
```

Comment:
@steveklabnik Is `nightly` correct for downloading from `HEAD`? I would think `dev` is fine there.